docs/guides/setup.html

```diff
@@ -18,13 +18,13 @@ <h2 id="toc_1">Step 1: Include the Video.js Javascript and CSS files in the head
 
 <p>You can download the Video.js source and host it on your own servers, or use the free CDN hosted version. As of Video.js 5.0, the source is <a href="http://babeljs.io/">transpiled from ES2015</a> (formerly known as ES6) to <a href="https://es5.github.io/">ES5</a>, but IE8 only supports ES3. In order to continue to support IE8, we&#39;ve bundled an <a href="https://github.com/es-shims/es5-shim">ES5 shim and sham</a> together and hosted it on the CDN.</p>
 
-<pre><code class="html">&lt;script src=&quot;//vjs.zencdn.net/ie8/1.1.0/videojs-ie8.min.js&quot;&gt;&lt;/script&gt;
+<pre><code class="html">&lt;script src=&quot;http://vjs.zencdn.net/ie8/1.1.0/videojs-ie8.min.js&quot;&gt;&lt;/script&gt;
```

Comment:
ok, I'll revert that

Gary Katsevman wrote:
>
> In docs/guides/setup.html
> <https://github.com/videojs/docs/pull/11#discussion_r44295258>:
>
> > @@ -18,13 +18,13 @@<h2 id="toc_1">Step 1: Include the Video.js 
> Javascript and CSS files in the head
> >
> > <p>You can download the Video.js source and host it on your own 
> servers, or use the free CDN hosted version. As of Video.js 5.0, the 
> source is<a href="http://babeljs.io/">transpiled from ES2015</a> 
> (formerly known as ES6) to<a href="https://es5.github.io/">ES5</a>, 
> but IE8 only supports ES3. In order to continue to support IE8, 
> we&#39;ve bundled an<a href="https://github.com/es-shims/es5-shim">ES5 
> shim and sham</a> together and hosted it on the CDN.</p>
> >
> > -<pre><code class="html">&lt;script 
> src=&quot;//vjs.zencdn.net/ie8/1.1.0/videojs-ie8.min.js&quot;&gt;&lt;/script&gt;
> > +<pre><code class="html">&lt;script 
> src=&quot;http://vjs.zencdn.net/ie8/1.1.0/videojs-ie8.min.js&quot;&gt;&lt;/script&gt;
>
> Just noticed this, but we actually do want the urls to be
> |//vjs.zencdn.net...|. Protocol relative urls are preferable for these
> kinds of things so that it'll get the content from either |http| or
> |https| depending on the parent page.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/videojs/docs/pull/11/files#r44295258>.